From 7a0340c57d5c7d23783096c174672eb502c6c2b3 Mon Sep 17 00:00:00 2001
From: Daniel van Vugt <daniel.van.vugt@canonical.com>
Date: Fri, 6 Sep 2019 17:54:00 +0800
Subject: [PATCH 19/30] kms-impl-device: Use portable formatting for 64-bit
 integers

On 64-bit systems uint64_t is %lu but on 32-bit systems it is %llu.

https://gitlab.gnome.org/GNOME/mutter/merge_requests/778
---
 src/backends/native/meta-kms-impl-device.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/backends/native/meta-kms-impl-device.c b/src/backends/native/meta-kms-impl-device.c
index 2f7cb7f89..dde192b42 100644
--- a/src/backends/native/meta-kms-impl-device.c
+++ b/src/backends/native/meta-kms-impl-device.c
@@ -266,7 +266,8 @@ get_plane_type (MetaKmsImplDevice       *impl_device,
     case DRM_PLANE_TYPE_OVERLAY:
       return META_KMS_PLANE_TYPE_OVERLAY;
     default:
-      g_warning ("Unhandled plane type %lu", props->prop_values[idx]);
+      g_warning ("Unhandled plane type %" G_GUINT64_FORMAT,
+                 props->prop_values[idx]);
       return -1;
     }
 }
-- 
2.23.0

